entity_version.post_update.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Entity version post updates.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Apply entity version settings for bundles with version field.
+ */
+function entity_version_post_update_configure_settings() {
+  // Introduced new configuration to select a version field of an entity bundle
+  // as the main version field to use by the (sub)module functionalities,
+  // instead of searching for all version fields on the bundle and using the
+  // first one.
```

Comment:
We shouldn't explain here that we introduced a new configuration, we should explain what we are trying to do. Also the function comment is too cryptic, we are not apply settings, we are insuring that the old behaviour is the same as the new one.
Let's resume everything with a single function comment: `Migrate to configuration-based version field settings.`